src/components/AppHeader.vue

```diff
@@ -1,3 +1,109 @@
+<script lang="ts" setup>
+import { computed, onMounted, ref, onUnmounted } from 'vue'
+import { useRoute, useRouter } from 'vue-router'
+import { useStore } from 'vuex'
+
+import AccountUserIcon from '@/assets/iconography/account-user.svg'
+import ArcanaLogo from '@/assets/iconography/arcana-dark-vertical.svg'
+import ArcanaFavicon from '@/assets/iconography/arcana-favicon.svg'
+import MenuIcon from '@/assets/iconography/menu.svg'
+import CloseIcon from '@/components/icons/CloseIcon.vue'
+import VHeader from '@/components/lib/VHeader/VHeader.vue'
+
+defineProps({
+  hideBanner: Boolean,
```

Comment:
Prefer the object syntax for defining props:

```suggestion
  hideBanner: {
    type: Boolean,
    required: false,
    default: false,
  }
```